platforms/web/public/locales/en/common.json

```diff
@@ -24,6 +24,9 @@
   "series": "series",
   "sign_in": "Sign in",
   "sign_up": "Sign up",
+  "slide_indicator": "Page {{page}} of {{pages}}",
   "slide_left": "Slide left",
+  "slide_next": "Next slide",
+  "slide_previous": "Previous slide",
```

Comment:
Shouldn't it be `Next page`?

Reply:
Meh... I'm hesitant about that... I know i use "pages" as announcement, but "Next page" sounds a little bit to drastically in my eyes. When using "slide" people will know they are operating a carousel

Reply:
Yes, but in case there are multiple items, I'm not sure if you can speak of going "one slide to the right". Just doesn't really feel right to me. However, just merge if you're sure about it!